CMakeLists.txt

```diff
@@ -27,6 +27,9 @@ if(POLICY CMP0056)
     cmake_policy(SET CMP0056 NEW)
 endif()
 
+# build the sdk targets
+project("aws-cpp-sdk-all" VERSION "${PROJECT_VERSION}" LANGUAGES CXX)
```

Comment:
project() defines the INSTALL_PREFIX which we test for later.
The call is intentionally placed *after* the cmake commands to build the third-party libs.
We should have documented that better.

Reply:
ping!
any chance of updating this PR?